brave/browser-laptop
# Add compact bravery panel with Aphrodite

Closes #8954: Add compact bravery panel with Aphrodite
Closes #8913: Refactor braveryPanel with Aphrodite
Closes #5726: The bravery panel should be much wider
Fixes #8953: Align 'Reload' text and icon
Fixes #8952: Set opacity to FormDropdown's wrappers and their titles
Fixes #8949: Rename 'general.blocked-count-badge' to 'shields.blocked-count-badge'
Addresses #7989: Refactor forms.less with Aphrodite
Addresses #7321: Refactor switchControl

Submitter Checklist:


**Test plan 1:**
1. Run tests locally; npm run test -- --grep='Bravery Panel'
2. Open about:preferences#shields
3. Enable the compact panel option
4. Run manual tests described here: https://github.com/brave/browser-laptop/wiki/Manual-Tests#bravery-settings

**Test Plan 2:**
1. Open about:preferences#shields
2. Make sure 'Display block count badge on shields button' works

**Test Plan 3:**
1. Open about:styles
2. Click "Dropdowns"
3. Make sure BraveryPanelDropdown is displayed

Reviewer Checklist:

Tests
